Assets/MixedRealityToolkit/Inspectors/Profiles/MixedRealitySceneSystemProfileInspector.cs

```diff
@@ -213,6 +213,9 @@ public override void OnInspectorGUI()
             });
 
             serializedObject.ApplyModifiedProperties();
+
+            // Keep this inspector perpetually refreshed
+            EditorUtility.SetDirty(target);
```

Comment:
>SetDirty [](start = 26, length = 8)

Just to check - this doesn't cause us to spike Unity CPU usage when this inspector is open right? I have flashbacks to the build window

Reply:
Not in this case but yes that can absolutely be a problem so this technique should be used sparingly.

Reply:
There was a report that using this constantly, even when changes aren't being made, marks the scene as changed and needing to be saved: https://github.com/microsoft/MixedRealityToolkit-Unity/issues/4516, which seems annoying

Reply:
In this case it only marks the profile as dirty and not the scene. It must be because it's a ScriptableObject asset and not an actual prefab.